PopMovies/PopMovies/Sources/Views/ListPopMovies/Views/PopMoviesCell.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  PopMoviesCell.swift
+//  PopMovies
+//
+//  Created by Angela Alves on 20/10/22.
+//
+
+import Foundation
+import Kingfisher
+import TinyConstraints
+import UIKit
+
+class PopMoviesCell: UICollectionViewCell {
+
+    static let identifer = "popMoviesCell"
+    private var movie: MovieItem?
+
+    // MARK: - Init
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        configureSubViews()
+        configureConstraints()
+        backgroundColor = Theme.currentTheme.color.cellColor.rawValue
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private let stackView = UIStackView() .. {
+        $0.axis = .vertical
+        $0.spacing = 8
+    }
+
+    private let imageView = UIImageView() .. {
+        $0.contentMode = .scaleAspectFit
+        $0.layer.shadowColor = UIColor.black.cgColor
+        $0.layer.shadowOpacity = 0.4
+    }
+
+    private let titleStackView = UIStackView() .. {
+        $0.axis = .horizontal
+    }
+
+    private let titleLabel = UILabel() .. {
+        $0.font = UIFont.systemFont(ofSize: 16)
+        $0.numberOfLines = 0
+    }
+
+    private let button = UIButton() .. {
+        $0.setContentHuggingPriority(.required, for: .horizontal)
+        $0.setCompressionResistance(.required, for: .horizontal)
+        let image = UIImage(systemName: "heart")!
```

Comment:
o método `setImage` aceita uma imagem opcional, não precisa fazer esse force unwrap (é sempre bom evitar)
```suggestion
        let image = UIImage(systemName: "heart")
```